### ISSUE TITLE
Improve a11y in the search experience

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1608]
Axe also complains about non-unique labels, but that might just be because their IDs are generated programatically? As far as I can see, they are unique

**Acceptance Criteria:**
- [ ] Wave suggests that the search input on the homepage/search page should have a <label>.  
    - Either provide a <label> or use the aria label by thing?
- [ ] Resolve Wave issues with KS filters on search page
    - I think this could be because the filter inputs are within the associated label tags?

**Comment by jack-skerrett-oak** [comment]
duplicate
